Door Detection

PepijnMC opened the pull request:
The two variations of doors are now detected and converted to an appropriate single segment.
![Dungeon Scrawl](https://user-images.githubusercontent.com/13134303/211189469-ba492c87-f1b4-4379-8982-dedfb01f23c0.png)
![Foundry](https://user-images.githubusercontent.com/13134303/211189470-57a3b919-d205-4ba7-84e5-e200392a9492.png)

PepijnMC committed:
Update DungeonScrawlImporterFormApplication.js

PepijnMC committed:
Implemented door detection and handling

Doors are now detected and properly converted to a single segment,

PepijnMC committed:
Update module.json

kid2407 commented:
That looks really nice! I'll have a look at the code when I have the time, but from a very quick scan seems quite decent.

PepijnMC committed:
Better documentation.

PepijnMC committed:
Reduce door variant 1 to a polyline

Both door variants now reduce to a single polyline, there's no need to use polygons for them and it saves a bit of processing later on.
